src/cljs/velho_ds/pages/main.cljs

```diff
@@ -14,146 +12,146 @@
-   [buttons/default-small {:content "Update"
-                           :icon "autorenew"
-                           :on-click-fn #(println "Small default button with icon and text clicked")}]
-   [buttons/default-small {:content "Update"
-                           :on-click-fn #(println "Small default button with text clicked")}]
-   [buttons/default-small {:icon "autorenew"
-                           :on-click-fn #(println "Small default button with icon clicked")}]
-
-   [buttons/primary-small {:content "Update"
-                           :icon "autorenew"
-                           :on-click-fn #(println "Small primary button with icon and text clicked")}]
-   [buttons/primary-small {:content "Update"
-                           :on-click-fn #(println "Small primary button with text clicked")}]
-   [buttons/primary-small {:icon "autorenew"
-                           :on-click-fn #(println "Small primary button with icon clicked")}]
-
-   [buttons/secondary-small {:content "Update"
-                             :icon "autorenew"
-                             :on-click-fn #(println "Small secondary button with icon and text clicked")}]
-   [buttons/secondary-small {:content "Update"
-                             :on-click-fn #(println "Small secondary button with text clicked")}]
-   [buttons/secondary-small {:icon "autorenew"
-                             :on-click-fn #(println "Small secondary button with icon clicked")}]
-
-   [buttons/outline-small {:content "Update"
-                           :icon "autorenew"
-                           :on-click-fn #(println "Small outline button with icon and text clicked")}]
-   [buttons/outline-small {:content "Update"
-                           :on-click-fn #(println "Small outline button with text clicked")}]
-   [buttons/outline-small {:icon "autorenew"
-                           :on-click-fn #(println "Small outline button with icon clicked")}]
-
-
-   [:h2 "Fields"]
-
-   [fields/keyvalue {:label "Title"
-                     :content "Value"}]
-
-   [fields/input-field {:label "Input with label and icon"
-                        :placeholder "Placeholder"
-                        :icon "search"
-                        :on-change-fn #(println %)
-                        :on-blur-fn #(println %)
-                        :icon-click-fn #(println (str "Icon Clicked"))}]
-
-   [fields/input-field {:placeholder "Placeholder"
-                        :icon "search"
-                        :icon-click-fn #(println (str "Icon Clicked"))}]
+   [:p.rds-quote "Fonts create hierarcies and purposeful alignments that guide users through the product and experience."]
 
-   [fields/input-field {:placeholder "Placeholder"}]
-
-   [fields/input-field {:label "Validation example"
-                        :content "Invalid value"
-                        :error-messages ["Value has to be valid!"]}]
+   [:h2.rds-header2 "Default fonts"]
+   [:h3.rds-header3 "Example of usage"]
+   [:div.rds-code-example
```

Comment:
If this div is needed with code examples I would prefer it to be introduced inside the macro-function so that a developer shouldn't have to wrap the component calls. That is to minimize the burden of using the code example. The aim for the code examples is that there is minimal effort from the developer to use it. If we make it easy/pleasing enough to use it will be used. Otherwise I'm not so sure if it will be.

Reply:
Also it seems like an extra repeating wrapper everywhere.